src/components/widgets-settings/forecast-payroll/forecast-payroll.directive.coffee

```diff
@@ -20,8 +30,10 @@ module.controller('SettingForecastPayrollCtrl', ($scope, ImpacDashboardsSvc) ->
     ImpacDashboardsSvc.load().then(
       (config) ->
         currentDashboard = config.currentDashboard
-        return unless w.metadata? && w.metadata.forecast_payroll?
-        $scope.selectedForecastPayroll = w.metadata.forecast_payroll
+        $scope.selectedForecastPayroll = {enable: false, time_sheet: false}
+        if w.metadata? && w.metadata.forecast_payroll? && typeof(w.metadata.forecast_payroll) == 'object'
+          $scope.selectedForecastPayroll = w.metadata.forecast_payroll
```

Comment:
```coffeescript
forecastPayroll = _.get(w, 'metadata.forecast_payroll')
$scope.selectedForecastPayroll = w.metadata.forecast_payroll if _.isPlainObject(forecastPayroll)
```

What are you trying to check with the `typeof` ? Are you trying to check for a plain object? Because that wont work. You can see [here](https://stackoverflow.com/questions/8511281/check-if-a-value-is-an-object-in-javascript) that it's a long subject hehe. But lucky lodash is to the rescue with [_.isPlainObject](https://lodash.com/docs/3.10.1#isPlainObject).